.github/workflows/terraform-staging.yml

```diff
@@ -51,6 +51,18 @@ jobs:
         run: echo "::set-output name=server_public_ip::$(cd terraform; terraform output server_public_ip)"
         if: success()
 
+      - name: Create .env file
+        env:
+          HOST_URL: ${{ steps.server_public_ip.outputs.server_public_ip }}
```

Comment:
This should be the domain name but I guess the IP could work?

Reply:
No idea if it will works! I will keep it like that at the moment and I will change it when we will be able to manage the sub-domains (as my main goal now is to be able to build the docker image in the terraform workflow)